qiskit_experiments/curve_analysis/standard_analysis/error_amplification_analysis.py

```diff
@@ -141,7 +141,9 @@ def _generate_fit_guesses(
         max_abs_y, _ = curve.guess.max_height(curve_data.y, absolute=True)
         max_y, min_y = np.max(curve_data.y), np.min(curve_data.y)
 
-        user_opt.bounds.set_if_empty(d_theta=(-np.pi, np.pi), base=(-max_abs_y, max_abs_y))
+        user_opt.bounds.set_if_empty(
+            d_theta=(-0.8 * np.pi, 0.8 * np.pi), base=(-max_abs_y, max_abs_y)
+        )
```

Comment:
Please update the docs to reflect this change. When you do so please also add a justification of why we use 0.8*pi as this seems a bit arbitrary. My understanding is that we want to avoid the points `+-pi` since they often correspond to symmetries of the fit functions.

Reply:
yes, this is correct. but this could allow 0.99999pi. I don't have justification for 0.8.

Reply:
0.8 looks okay to me.